src/components/sortable/helpers.ts

```diff
@@ -0,0 +1,17 @@
+import type { XYCoord } from 'react-dnd';
+
+export const getItemStyles = (clientOffset: XYCoord | null): React.CSSProperties => {
```

Comment:
Let's desctucture it from React import
```suggestion
export const getItemStyles = (clientOffset: XYCoord | null): CSSProperties => {
```

Reply:
Done, thanks!